enacts/enactstozarr.py

```diff
@@ -11,34 +11,22 @@
 
 CONFIG = pingrid.load_config(os.environ["CONFIG"])
 
-RR_MRG_NC_PATH = CONFIG["onset"]["rr_mrg_nc_path"]
-RR_MRG_ZARR_PATH = CONFIG["onset"]["rr_mrg_zarr_path"]
-RESOLUTION = CONFIG["onset"]["rr_mrg_resolution"]
-CHUNKS = CONFIG["onset"]["rr_mrg_chunks"]
-
-#Read daily files of daily rainfall data
-#Concatenate them against added time dim made up from filenames
-
-RR_MRG_PATH = Path(RR_MRG_NC_PATH)
-RR_MRG_FILE = list(sorted(RR_MRG_PATH.glob("*.nc")))
+ZARR_RESOLUTION = CONFIG["zarr_resolution"]
 
 def set_up_dims(xda):
+    
     datestr = Path(xda.encoding["source"]).name.split("_")[2]
     xda = xda.expand_dims(T = [dt.datetime(
       int(datestr[0:4]),
       int(datestr[4:6]),
       int(datestr[6:8])
     )])
     xda = xda.rename({'Lon': 'X','Lat': 'Y'})
+    
     return xda
 
-rr_mrg = xr.open_mfdataset(
-    RR_MRG_FILE,
-    preprocess = set_up_dims,
-    parallel=False
-).precip
-
-if not np.isclose(rr_mrg['X'][1] - rr_mrg['X'][0], RESOLUTION):
+def regridding(data, resolution):
```

Comment:
For the record, I would create an issue about regridding, unless already there but I don't think so, including the TODO comment below and including the desire to generalize to other dimensions.

Reply:
an issue for improving the method used for the regridding?

Reply:
yes. so that it is on the "issues" radar and so that whoever is going to tackle it is going to be reminded there is this piece of code here.